vmdb/app/models/miq_provision_microsoft/placement.rb

```diff
@@ -0,0 +1,34 @@
+module MiqProvisionMicrosoft::Placement
+  protected
+
+  def placement
+    log_header = "MIQ(#{self.class.name}.placement)"
+    desired    = manual_placement
+    [:host, :storage].each do |key|
+      object = desired[key]
+      next if object.nil?
+      $log.info("#{log_header} Using #{key.to_s.titleize} Id: [#{object.id}], Name: [#{object.name}]")
+      options["dest_#{key}".to_sym] = [object.id, object.name]
+    end
+  end
+
+  private
+
+  def manual_placement
+    update_placement_info
+  end
+
+  def selected_placement_host
+    Host.where(:id => get_option(:placement_host_name)).first
+  end
```

Comment:
My fault, this is needed in the current configuration.  @bronaghs and I discussed creating an infra layer in this modeling so more methods/state-machine steps can be shared between RedHat/VMware and MS.  That's a refactoring for a separate PR.  This is ok for now.